src/main/java/com/uber/jenkins/phabricator/coverage/CoberturaXMLParser.java

```diff
@@ -118,6 +118,21 @@ public InputSource resolveEntity(String publicId, String systemId) throws SAXExc
             NodeList classes = entry.getKey();
             List<String> sourceDirs = entry.getValue();
 
+            // Collect all filenames in coverage report
+            List<String> fileNames = new LinkedList<String>();
```

Comment:
Drive-by comment: from what I've read recently, either `ArrayList` or `ArrayDeque` is almost always better than `LinkedList`; this is now [checked by ErrorProne](http://errorprone.info/bugpattern/JdkObsolete)

Reply:
Thanks, fixed